app/server/routes/core.ts

```diff
@@ -1,12 +1,14 @@
-import { Response, Router } from "express";
+import { Request, Response, Router } from "express";
+import { memoisedApiGatewayAuthForHealthcheck } from "../apiGatewayDiscovery";
 import { withIdentity } from "../middleware/identityMiddleware";
 
 const router = Router();
 
 router.get(
   "/_healthcheck",
   withIdentity(200), // healthcheck needs identity redirect service to be accessible (returns 200 if redirect required)
-  (_, res: Response) => {
+  memoisedApiGatewayAuthForHealthcheck(),
```

Comment:
Just checking that express does definitely allow for an async function to be provided in the router chain?

Reply:
Seems like it: https://itnext.io/using-async-await-to-write-cleaner-route-handlers-7fc1d91b220b